keyboards/idank/sweeq/info.json

```diff
@@ -0,0 +1,80 @@
+{
+    "url": "https://github.com/idank/keyboards/tree/main/sweeq",
+    "maintainer": "idank",
+    "keyboard_name": "Sweeq",
+    "manufacturer": "idank",
+    "development_board": "promicro",
+    "usb": {
+        "vid": "0xC3AB",
+        "pid": "0x3139",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": [ "F4", "F5", "F6", "F7", "B1" ],
+        "rows": [ "D4", "C6", "D7", "E6" ]
+    },
+    "diode_direction": "COL2ROW",
+    "split": {
+        "enabled": true,
+        "soft_serial_pin": "D2"
+    },
+    "community_layouts": ["split_3x5_2"],
+    "layout_aliases": {
+        "LAYOUT": "LAYOUT_split_3x5_2"
+    },
+    "layouts": {
+        "LAYOUT_split_3x5_2": {
```

Comment:
There is a lot of weird alignment here for the y axis.  Is this intentional?

Reply:
It's copy pasted from sweep: https://github.com/qmk/qmk_firmware/blob/master/keyboards/ferris/sweep/info.json.